tests/test_parsing.js

```diff
@@ -12,1391 +12,1483 @@ const {
 } = SyntaxType;
 
 describe('Parser', function() {
-  it('should return a type name node when "TypeName" arrived', function() {
-    const typeExprStr = 'TypeName';
-    const node = Parser.parse(typeExprStr);
+  describe('Primitive types', function () {
```

Comment:
I believe that this should be&nbsp;done in&nbsp;a&nbsp;separate&nbsp;PR (so&nbsp;that this&nbsp;PR just focuses on&nbsp;one&nbsp;thing).

Reply:
I've removed the testing changes in this PR.